File: rb/increasing_sequence.rb
def sequence(array)
  sequence_length = 1
  sequences = []
  array.each_with_index do |e, i|
    if i > 0 && array[i] - array[i-1] >= 0
      sequence_length += 1
    elsif i > 0
      sequences << sequence_length
      sequence_length = 1
    end
  end

  sequences.max
end

File: js/increasing_sequence.js
var sequence = function(array) {
	var sequenceLength = 1;
	var sequenceLengths = [];
	for (var i = 1; i < array.length; i++) {
		if (array[i]-array[i-1] >= 0) {
			sequenceLength++;
		} else {
			sequenceLengths.push(sequenceLength);
			sequenceLength = 1;
		};
	};
	return Math.max.apply(null, sequenceLengths);
};